File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/IsFinite.java
package com.onevgo.functions;

public class IsFinite {
    public static void main(String[] args) {
        System.out.println(Double.isFinite(Math.sqrt(2)));
        System.out.println(Double.isFinite(Math.log(0)));
        System.out.println(Double.isFinite(Math.asin(2)));
    }
}

File: 01_Language/01_Functions/php/is_finite.php
<?php

echo sqrt(2)."\n";
var_dump(is_finite(sqrt(2))); // true

echo log(0)."\n"; // -INF
var_dump(is_finite(log(0))); // false

echo asin(2)."\n"; // NAN
var_dump(is_finite(asin(2))); // false

var_dump(is_finite(null)); // true

